jwst/cube_build/ifu_cube.py

```diff
@@ -683,51 +685,48 @@ def build_ifucube(self):
-                                                              beta_resol,
-                                                              wave_resol,
-                                                              self.naxis1, self.naxis2, self.naxis3,
-                                                              self.xcenters, self.ycenters, self.zcoord,
-                                                              self.spaxel_flux,
-                                                              self.spaxel_weight,
-                                                              self.spaxel_iflux,
-                                                              self.spaxel_var,
-                                                              spaxel_alpha, spaxel_beta, spaxel_wave,
-                                                              flux,
-                                                              err,
-                                                              coord1, coord2, wave,
-                                                              alpha_det, beta_det,
-                                                              'emsm',
-                                                              rois_pixel, roiw_pixel,
-                                                              weight_pixel,
-                                                              softrad_pixel,
-                                                              scalerad_pixel)
-                # --------------------------------------------------------------------------------
-                # AREA - 2d method only works for single files local slicer plane (internal_cal)
-                # --------------------------------------------------------------------------------
-                elif self.interpolation == 'area':
-                    with datamodels.IFUImageModel(ifile) as input_model:
-
+                        wave_resol = self.instrument_info.Get_RP_ave_Wave(this_par1,
+                                                                          this_par2)
+
+                        alpha_resol = self.instrument_info.Get_psf_alpha_parameters()
+                        beta_resol = self.instrument_info.Get_psf_beta_parameters()
+                        wcsobj = input_model.meta.wcs
+                        worldtov23 = wcsobj.get_transform(wcsobj.output_frame.name, "v2v3")
+                        v2ab_transform = wcsobj.get_transform('v2v3', 'alpha_beta')
+
+                        spaxel_v2, spaxel_v3, zl = worldtov23(spaxel_ra,
+                                                              spaxel_dec,
+                                                              spaxel_wave)
+
+                        spaxel_alpha, spaxel_beta, spaxel_wave = v2ab_transform(spaxel_v2,
+                                                                                spaxel_v3,
+                                                                                zl)
```

Comment:
I have talked to D Law and we both agree that we can get rid of the self.weighting = miripsf. It was a plan long ago that was thought to be needed. But it has not been tested and it looks like it will not be used. So I am going to delete all this code. I can do it with this PR or a separate pr

Reply:
A separate PR is better.